bcgov/mds
# Mds 3714

# Main

- Able to update the mine list of a minespace user in Core-web
 - this includes being able to add new mines and delete existing mines

# Other


- When updating a proponent, the pop-up will read Update User and Edit Proponent to correctly represent the form as updating the user. 



# How to test

- Create 2-3 mines
- Create multiple proponents in Core (Navigate to Admin -> MineSpace Management -> Users -> Create Proponent)
- Add and delete mines across the proponents you have created
- Data should update in real-time without the need for a hard refresh

# Notes

- N/A
